pkg/config/config.go

```diff
@@ -151,7 +151,14 @@ type Config struct {
 
 // Environ returns the config as a list of environment variables.
 func (c *Config) Environ() []string {
-	envs := []string{}
+	ex, err := os.Executable()
+	if err != nil {
+		ex = "soft-serve"
+	}
+	ex = filepath.ToSlash(ex)
+	envs := []string{
+		fmt.Sprintf("SOFT_SERVE_EXE_PATH=%s", ex),
```

Comment:
IMO, `BIN_PATH` makes more sense as `EXE_PATH` might imply Windows `.exe` extension file path
```suggestion
		fmt.Sprintf("SOFT_SERVE_BIN_PATH=%s", ex),
```